src/components/modal/Modal.scss

```diff
@@ -133,3 +135,33 @@
     z-index: 89;
     display: flex;
 }
+
+//Responsive support
+.is-responsive-web {
+    .modal {
+        @include breakpoint($modal-fullscreen) {
+            padding: 0;
+        }
+    }
+
+    .modal-dialog-container {
+        width: 100%;
+        margin: auto;
+
+        @include breakpoint($modal-fullscreen) {
+            height: 100%;
```

Comment:
Percentage heights are fragile, since they're relative to their direct parent. In this case, `.modal` is a flex container, so this should be equivalent to `flex-basis: 100%`.  I'd suggest testing these changes in IE11 and Safari to ensure they work properly, since those browsers can be a little touchy with flexbox.

Reply:
Updated this to use `flex-basis: 100%` instead. Seems to be okay on other browsers.